ibm_catalog.json

```diff
@@ -58,14 +58,14 @@
 			"flavors": [
 				{
 					"label": "Basic (Deploy on Code Engine)",
-					"name": "Code-Engine-d7b9",
+					"name": "basic",
```

Comment:
I guess that renaming will not have any impact as well?

Reply:
https://github.com/terraform-ibm-modules/stack-retrieval-augmented-generation/pull/197#discussion_r1781425150